### PR TITLE
Add support for m1 - (Darwin+arm64)

### PR DESCRIPTION
**Pull request by mikeknox**
- Build arm64 binaries
- include Darwin arm64 release in krew.yaml
- enable prerelease support for goreleaser

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ release:
   github:
     owner: mattfenwick
     name: kubectl-cyclonus
+  prerelease: auto
```
Comment (mattfenwick) [inline .goreleaser.yml:6]: ahhh nice 👍 wasn't aware of this before

just looked at https://goreleaser.com/customization/release/ , my understanding of this field is that it `auto` is a better setting than the default of `false` -- is that accurate ?

thanks for fixing this !

Comment (mattfenwick) [inline .goreleaser.yml:6]: ahhh nice 👍 wasn't aware of this before

just looked at https://goreleaser.com/customization/release/ , my understanding of this field is that it `auto` is a better setting than the default of `false` -- is that accurate ?

thanks for fixing this !

**Comment by mattfenwick** [comment]
hi @mikeknox thanks for putting this together !

I'm not sure how to test this but happy to merge if you think it's already good to go -- lmk !

**Comment by mattfenwick** [comment]
hi @mikeknox thanks for putting this together !

I'm not sure how to test this but happy to merge if you think it's already good to go -- lmk !

**Comment by mattfenwick** [comment]
oh is github broken and doubling my comments everywhere ?

File: .goreleaser.yml (hunk 1 above)
Comment (mikeknox) [reply .goreleaser.yml:6]: @mattfenwick I think so, from my limited reading / experimentation with it.

**Comment by mikeknox** [comment]
@mattfenwick , I tested it as best I could on my fork ... should be good to merge

**Comment by mattfenwick** [comment]
thanks @mikeknox !

merged, and https://github.com/mattfenwick/kubectl-cyclonus/releases/tag/v0.1.6 is on its way -- lmk if that works for you !

**Comment by mikeknox** [comment]
👍 @mattfenwick , I was able to install on an m1 using the manifest that's been submitted to krew-index.

**Comment by mattfenwick** [comment]
Awesome!!  Thanks again for the patch :smile:

**Comment by dylanANZ** [comment]
I'm seeing this error, any tips?

```failed to install some plugins: [cyclonus]: plugin "cyclonus" does not offer installation for this platform```

**Pull request merged**